### PR TITLE
Login menu update

### PR DESCRIPTION
**Pull request by ccnPeon**
Lab guides updated to reflect new menu as well.

### CODE REVIEW

**Comment by networkRob** [comment]
Currently reviewing the files, mentioned to @ccnPeon that the `default.yaml` file should contain a pointer to the desired yaml file that should be default. That way there aren't multiple copies of the same lab to maintain. He is working on that now.

File: topologies/beta-datacenter/files/.screenrc [hunk 1]
```diff
@@ -17,3 +17,4 @@ screen 6 ssh 192.168.0.17
 screen 7 ssh 192.168.0.31
 screen 8 ssh 192.168.0.32
 screen 9 ssh 192.168.0.44
+screen 10 /usr/local/bin/login.py
```
Comment (networkRob) [inline topologies/beta-datacenter/files/.screenrc:20]: Add carriage return
```suggestion
screen 10 /usr/local/bin/login.py

```

**Comment by ccnPeon** [comment]
Added logic and updated yaml files to point to default file rather than have copy of menus for default menu.

File: topologies/datacenter/labguides/source/cvx.rst [hunk 2]
```diff
@@ -47,7 +47,7 @@ CVX
             interface Ethernet4
               description HOST2
               channel-group 4 mode active
-              lacp rate fast
+              lacp timer fast
```
Comment (networkRob) [inline topologies/datacenter/labguides/source/cvx.rst:50]: This topo used the old version of EOS, so probably not change the CLI syntax on this topo:
```suggestion
              lacp rate fast
```

File: topologies/datacenter/labguides/source/l2evpn.rst [hunk 3]
```diff
@@ -42,7 +42,7 @@ L2 EVPN
         !
         interface Ethernet4
           channel-group 4 mode active
-          lacp rate fast
+          lacp timer fast
```
Comment (networkRob) [inline topologies/datacenter/labguides/source/l2evpn.rst:45]:
```suggestion
          lacp rate fast
```

**Comment by networkRob** [comment]
@ccnPeon SFTP it broken with the new login menu/script. This functionality does need to be present as it is used by SEs and the training team. Here is the output of sftp'ing to an instance.

```
(virtual) robmartin:atd-public robmartin$ sftp arista@X.X.X.X
arista@X.X.X.X's password:
Received message too long 168438314
Ensure the remote shell produces no output for non-interactive sessions.
(virtual) robmartin:atd-public robmartin$
```

**Comment by ccnPeon** [comment]
What topology did you test with? I just tested with DC-Latest with the new menu and this worked fine:

```arista@ip-10-33-5-158:~$ sftp arista@192.168.0.16
Connected to 192.168.0.16.
sftp>```

**Pull request merged**